Source/WebCore/css/CSSSelector.h

```diff
@@ -573,10 +524,8 @@ inline CSSSelector::CSSSelector(const CSSSelector& o)
 
 inline CSSSelector::~CSSSelector()
 {
-    ASSERT_WITH_SECURITY_IMPLICATION(!m_destructorHasBeenCalled);
-#if !ASSERT_WITH_SECURITY_IMPLICATION_DISABLED
+    RELEASE_ASSERT_WITH_SECURITY_IMPLICATION(!m_destructorHasBeenCalled);
```

Comment:
Hm... we're making this a release assert.
Perhaps we should do it in a separate patch?

Reply:
The style checker script says all `ASSERT_WITH_SECURITY_IMPLICATION` should be changed to `RELEASE_ASSERT_WITH_SECURITY_IMPLICATION`.

My choices were:

1. Ignore the style checker, and upload a pull request that fails the style checker.
2. Try to rearrange the pull request so it doesn’t touch those lines.
3. Remove the destruction assertion entirely.
4. Convert the destruction assertion into a release assertion.
5. Do a separate pull request *first* with one of the above changes.

After considering all those options I went with the above. Do you have a different preference?

Reply:
Okay. Yeah, I usually ignore that style checker error when I'm dealing with an existing debug only assertion.